### PR TITLE
window-rules: add `fixedPosition` property

### PR DESCRIPTION
**Pull request by johanmalm**

### CODE REVIEW

File: docs/labwc-config.5.scd [hunk 1]
```diff
@@ -526,6 +526,10 @@ situation.
 *<windowRules><windowRule ignoreFocusRequest="">* [yes|no|default]
 	*ignoreFocusRequest* prevent window to activate itself.
 
+*<windowRules><windowRule ignoreUsableArea="">* [yes|no|default]
+	*ignoreUsableArea* do not re-position on changing usable area (which
+	takes into account exclusive layer-shell clients)
```
Comment (Consolatis) [inline docs/labwc-config.5.scd:531]: Suggest adding something like "and `<margin>` settings"

**Comment by Consolatis** [comment]
Other than the small docs nitpick this LGTM. Didn't test.

**Comment by johanmalm** [comment]
Thanks for review. Will incorporated.

How about the following two changes too:

1. `s/ignoreUsableArea/fixPosition/`
2. Disallow alt+mouse-press to move/resize when this property == true

**Comment by stefonarch** [comment]
> Disallow alt+mouse-press to move/resize when this property == true

Good idea!

**Comment by Consolatis** [comment]
> Thanks for review. Will incorporated.
> 
> How about the following two changes too:
> 
> 1. `s/ignoreUsableArea/fixPosition/`
> 2. Disallow alt+mouse-press to move/resize when this property == true

Sounds good to me.

**Comment by Consolatis** [comment]
:)

**Pull request merged**